Snakefile-proteomics

```diff
@@ -38,11 +38,21 @@ rule check_pride_tsv_files:
 
         source {workflow.basedir}/bin/proteomics_functions.sh
 
-        # currently this does not exit if the files do not exist
-        check_tsv_data_file_not_empty_of_data {wildcards.accession}.tsv.undecorated 2
-        check_tsv_data_file_not_empty_of_data {wildcards.accession}.tsv.undecorated.aggregated 2
-        check_tsv_data_file_not_empty_of_data {wildcards.accession}.tsv 3
-
+        if [ {params.is_old} == "False" ]; then
+            # currently check_tsv_data_file_not_empty_of_data does not exit if the files do not exist
+            if [ -e {wildcards.accession}.tsv.undecorated ] && [ -e {wildcards.accession}.tsv.undecorated.aggregated ] && [ -e {wildcards.accession}.tsv ]; then
+                check_tsv_data_file_not_empty_of_data {wildcards.accession}.tsv.undecorated 2
+                check_tsv_data_file_not_empty_of_data {wildcards.accession}.tsv.undecorated.aggregated 2
+                check_tsv_data_file_not_empty_of_data {wildcards.accession}.tsv 3
+            else
+                echo "Starting files do not exist for proteomics experiment {wildcards.accession} "
```

Comment:
This works well but might good to state which file is missing..

I know I am asking for too much..